packages/integration-tests/src/test-e2e/create_amplify.test.ts

```diff
@@ -32,11 +32,19 @@ void describe(
       // Force 'create-amplify' installation in npx cache by executing help command
       // before tests run. Otherwise, installing 'create-amplify' concurrently
       // may lead to race conditions and corrupted npx cache.
-      await execa('npm', ['create', amplifyAtTag, '--yes', '--', '--help'], {
-        // Command must run outside of 'amplify-backend' workspace.
-        cwd: os.homedir(),
-        stdio: 'inherit',
-      });
+      const output = await execa(
+        'npm',
+        ['create', amplifyAtTag, '--yes', '--', '--help'],
+        {
+          // Command must run outside of 'amplify-backend' workspace.
+          cwd: os.homedir(),
+        }
+      );
+
+      assert.match(output.stdout, /--help/);
+      assert.match(output.stdout, /--version/);
+      assert.match(output.stdout, /Show version number/);
+      assert.match(output.stdout, /--yes/);
```

Comment:
nit: assertion should be in `it` block. which means we'd call `--help` in two places for two different reasons:
1. in before in order to hydrate npm cache
2. in additional `it('should print help', ...)` in order to validate that help actually works.

Reply:
Why not saving some time by only running it once?

Reply:
readability.

assertions placed in distinct named test are easier to discover for future reader and it's easier to understand intent.
therefore I think a chance that somebody accidentally deletes this coverage is higher if this code is blended with setup code (person editing setup code my not realize importance of these assertions).